### DISCUSSION TITLE
Case opening with inventory changer.

### DISCUSSION THREAD

**Discussion by IlluminumCS** [danielkrupinski/Osiris#3508] [Q&A] [answered]
Can you get a knife from the cases you open with inventory changer? Ive opened over a 1000 cases and havent gotten one.

**Comment by danielkrupinski** [comment, marked as answer]
Currently you can't.

**Reply by IlluminumCS** [reply]
is there any info on when it would be out? and could the chances stay the same as normal unboxing?

**Comment by Sumandora** [comment]
> Currently you can't.

Could you explain why? Why wouldn't it be possible? Are you modifying anything related to that?
